extensions/wikia/Recirculation/js/trackers/liftigniter.js

```diff
@@ -17,3 +17,4 @@ $p("init", "l9ehhrb6mtv75bp2", {
     }
 });
 $p("send", "pageview");
+$p("setRequestFields", ["rank", "thumbnail", "title", "url", "presented_by", "author"]);
```

Comment:
It seems odd to have this call along side the tracking code. Was it not working properly in the helper?